src/openrct2/object/MusicObject.cpp

```diff
@@ -80,6 +80,13 @@ void MusicObject::Load()
 void MusicObject::Unload()
 {
     LanguageFreeObjectString(NameStringId);
+    if (_hasPreview)
```

Comment:
You can probably omit this check.

Reply:
Yeah its safe to call with zero image and it is allocated on L77 unconditionally